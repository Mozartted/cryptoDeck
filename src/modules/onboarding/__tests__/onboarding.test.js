import React from 'react';
import {shallow} from 'enzyme';
// import Button from './Button';
import Onboarding from "../views/layouts/onboarding"
describe('Onboarding Screens', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const component = shallow(<Onboarding/>)
            expect(component).toMatchSnapshot()
        });
    });
});