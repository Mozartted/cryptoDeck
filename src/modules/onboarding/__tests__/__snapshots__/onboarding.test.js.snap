// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Onboarding Screens Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
